### DISCUSSION TITLE
Astroid Plugin with enable bootstrap js bad google pagespeed -

### DISCUSSION THREAD

**Discussion by WM-Loose** [templaza/astroid-framework#392] [Q&A]
Hello Sonny, today we discovered that if you deactivate "bootstrat js" in the astroid plugin, the google pagespeed (mobile) values improve by 50%. 

As soon as bootstrab js is activated again, the page speed goes down. I realize that bootstrap is loaded from cdn and cannot be loaded locally via the plugin but the fact that it costs so much speed is not good. Is there a way to solve the problem? 

Greetings Dirk

**Comment by sonvnn** [comment]
It's Bootstrap lib from Joomla Core. I will consider another way in Astroid 3.

Thanks for feedback :)

**Comment by WM-Loose** [comment]
That are great news.
Many thanks for your response.
